src/main/java/com/hackaton/model/TemporaryModelClass.java

```diff
@@ -0,0 +1,25 @@
+package com.hackaton.model;
+
+import com.hackaton.interfaces.ResultInterface;
+
+//TODO usunąć klasę
```

Comment:
Czemu klasa do usunięcia? Jako przykladowy model do przechowywania danych odnośnie zadania Must Have jest ok. 

Tzn - jak encja. Bo do łatwiejszego zliczania potrzeba czegoś, co będzie zbudowane na kształt słownika (dictionary). Przynajmniej tak mi się wydaje

Reply:
Ta klasa została inaczej przeze mnie teraz nazwana. Arkowi potrzeba tylko 2 gety i je dostanie od manipulatora mojego. Za ok 15 minut powinien być manipulator gotowy

Reply:
Manipulatora? A nie modelu?

Reply:
Może być jakakolwiek nowa klasa. Jeśli implementuje mój interfejs z dwoma getterami to zadziała.

Reply:
I ok.